### PR TITLE
Add EnableSyncAdminReports

### PR DESCRIPTION
**Pull request by peterlewis**
Add EnableSyncAdminReports as documented in https://docs.microsoft.com/en-us/onedrive/sync-health?tabs=macos

### CODE REVIEW

File: OneDrive/deploy-and-configure-on-macos.md [hunk 1]
```diff
@@ -250,6 +251,16 @@ The example for this setting in the .plist file is:
 \<string\>(Keyword such as *.PST)\</string\><br/>
 \</array\><br/>
 
+### EnableSyncAdminReports (Preview)
+
+<a name="EnableSyncAdminReports"> </a>
+
+This setting specifies whether Sync Admin Reports is enabled.
+
+If you enable this setting, OneDrive sync health reports will be sent to OneDrive sync health dashboard in the Microsoft 365 Apps Admin Center.
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:260]:
```suggestion
If you enable this setting, OneDrive sync health reports will be sent to OneDrive sync health dashboard in the Microsoft 365 Apps admin center.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @peterlewis, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@mkbond007 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by mkbond007** [comment]
@cartergreen Can you please review?

**Comment by MikePlumleyMSFT** [comment]
Covered in [OneDrive sync reports in the Apps Admin Center](https://learn.microsoft.com/sharepoint/sync-health)

**Comment by peterlewis** [comment]
@MikePlumleyMSFT I appreciate it is covered in [OneDrive sync reports in the Apps Admin Center](https://learn.microsoft.com/sharepoint/sync-health), but given that the article contains a "[List of settings](https://learn.microsoft.com/en-us/sharepoint/deploy-and-configure-on-macos#list-of-settings)" I feel that it makes sense that the `EnableSyncAdminReports' setting should also be mentioned, even if it's just with a link to the aforementioned.

**Comment by MikePlumleyMSFT** [comment]
I've added it to the related topics list at the bottom of the article.

**Comment by scanum** [comment]
Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**